### PR TITLE
Yang bam2.0 add borker property to trigger stats update

### PR DESCRIPTION
**Pull request by YangQi007**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -357,11 +357,11 @@ <h1>bam2.iobio</h1>
       });
 
       document.addEventListener('bed-file-selected', (event) => {
-        console.log(event.detail)
+        broker.bedText = event.detail.bedText;
```
Comment (anderspitman) [inline index.html:360]: This needs to go in bam.iobio/index.html right? This index.html needs to be removed, as it can create confusion.

**Review by anderspitman — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (YangQi007) [reply index.html:360]: I think so. I forgot why I kept the index.html in the `iobio-charts` repo.

**Comment by anderspitman** [comment]
@YangQi007 it's better not to combine multiple tasks in a single PR. Sorry I should have clarified that removing index.html should be done in a separate PR. It's probably more trouble than it's worth to undo it now, but just for future reference.

I'm not sure what's causing the conflicts though. Can you try merging main from GitHub and see if that fixes it?

**Comment by YangQi007** [comment]
> @YangQi007 it's better not to combine multiple tasks in a single PR. Sorry I should have clarified that removing index.html should be done in a separate PR. It's probably more trouble than it's worth to undo it now, but just for future reference.
> 
> I'm not sure what's causing the conflicts though. Can you try merging main from GitHub and see if that fixes it?

No, my bad. I should have thought about that. I just reverted that commit. Now the conflict should be resolved.

**Review by anderspitman — APPROVED** [review]

**Comment by anderspitman** [comment]
Easy enough, thanks!

**Pull request merged**